configure.ac

```diff
@@ -4,7 +4,7 @@
 # make sure we're interpreted by some minimal autoconf
 AC_PREREQ(2.57)
 
-AC_INIT(sparsehash, 2.0.2, google-sparsehash@googlegroups.com)
+AC_INIT(sparsehash, 2.0.3, google-sparsehash@googlegroups.com)
```

Comment:
I`m not sure this is sufficient, with these changes locally after I run ./configure I still get 
```
    /* Define to the full name and version of this package. */
    #define PACKAGE_STRING "sparsehash 2.0.2"
    /* Define to the version of this package. */
    #define PACKAGE_VERSION "2.0.2"
```
in config.h